pkg/db/db.go

```diff
@@ -0,0 +1,42 @@
+package db
+
+import (
+	"database/sql"
+	"os"
+
+	_ "modernc.org/sqlite"
+)
+
+var db *sql.DB
+
+const schema = `
+CREATE TABLE IF NOT EXISTS scheduler (
+    id INTEGER PRIMARY KEY AUTOINCREMENT,
+    date CHAR(8) NOT NULL DEFAULT "",
+    title VARCHAR(255) NOT NULL,
+    comment TEXT,
+    repeat VARCHAR(128) NOT NULL DEFAULT ""
+);
+CREATE INDEX IF NOT EXISTS date_idx ON scheduler (date);
+`
+
+// Инициализируем соединение с бд и создаем таблицы если их нет
+func Init(dbFile string) (err error) {
+	_, err = os.Stat(dbFile)
+	needsInstall := os.IsNotExist(err)
+
+	db, err = sql.Open("sqlite", dbFile)
+	if err != nil {
+		return err
```

Comment:
лучших всегда оборачивать ошибку.
```suggestion
		return fmt.Errorf("cannot open database: %w", err)
```